### PR TITLE
Prepare for 1.10.0 Release

### PR DESCRIPTION
**Pull request by barrettk**
Changeset: https://github.com/metrumresearchgroup/bbr/compare/1.9.0...release/1.10.0

### CODE REVIEW

**Comment by barrettk** [comment]
### Scorecard Details

image

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  logic can be implemented for derived model types, such as the `nmbayes` model
+  type in `bbr.bayes`. (#663)
+
+* The character methods for `tail_lst()` and `tail_output()` have been adjusted
+  to work with custom model types where `*.lst` and `OUTPUT` files do not reside
+  in the top-level output directory. (#661)
+  
+* `get_data_path()` can now pull from the control stream file if the model
+  has not yet been submitted. This allows functions like `nm_data()` to be used
+  pre-model submission. (#664)
```
Comment (kyleam) [inline NEWS.md:19]: Do you think it's worth mentioning

 * that there's no a `.check_exists` argument that defaults to `FALSE`
 * support for log (via new log methods)
?

**Review by kyleam — APPROVED** [review]
Looks good.  I've left a couple of suggestions for other things to mention in the NEWS, if you agree.

I've also verified that `check_pkgdown()` is happy.

File: NEWS.md (hunk 1 above)
Comment (barrettk) [reply NEWS.md:19]: Yeah thought about this. Was worried the bullet would be too long, but will add those elements

Comment (kyleam) [reply NEWS.md:19]: > Was worried the bullet would be too long

Up to you, but you can put them as separate bullets.

Comment (barrettk) [reply NEWS.md]: @kyleam let me know what you think of the new bullet point before I merge

Comment (kyleam) [reply NEWS.md]: Works for me.  Thanks for adding.

**Pull request merged**